packages/generator-common/src/package-description.ts

```diff
@@ -0,0 +1,10 @@
+/**
+ * @internal
+ */
+export function packageDescription(
+  packageName: string,
+  formatter?: (packageName: string) => string
+): string {
+  const formattedPackageName = formatter ? formatter(packageName) : packageName;
+  return `SAP Cloud SDK for JavaScript: Generated client for service ${formattedPackageName}`;
```

Comment:
[req]
@FrankEssenberger I currently find this confusing because in the service mapping we make a difference between service name (e.g. "Business Partner Service") and npm package name (e.g. "@sap/cloud-sdk-vdm-business-partner-service").
This seems to use the package name, which is incorrect.

Reply:
Also, when I look further into the code it seems that this is even derived from the directory name, not the service name nor package name, which I think mixes even more things in here.

Reply:
package generator and openapi-generator have readme.ts that make README.md. The service name that is retrieved by readme.ts is there in the first line of it. [`VdmServiceMetadata.speakingModuleName`](https://github.com/SAP/cloud-sdk-js/blob/8678c543f5193df838d4b440f232598b2a911a6f/packages/generator/src/service/readme.ts#L15)  and [`openApiDocument.serviceName`](https://github.com/SAP/cloud-sdk-js/blob/8678c543f5193df838d4b440f232598b2a911a6f/packages/openapi-generator/src/file-serializer/readme.ts#L14) are used as the service name in the readme.ts. I only use these 2 for the service name in the package.json description, not `packageName` or `directoryName` as before.